src/custom/estate/__manifest__.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "estate",
+
+    'summary': """
```

Comment:
Better also learn to add summary. Same for description

Reply:
Noted.